### ISSUE TITLE
Update unity package

### ISSUE THREAD

**Issue by ulisesyo** [camsjams/i18n-unity-csharp#6]
The unity package which is in the root of the project has still sealed the class, so it is out of sync with documentation

**Comment by GabLeRoux** [comment]
I agree with you. This should be automated and the `unitypackage` should be provided through [Releases section](https://github.com/MoonGateLabs/i18n-unity-csharp/releases) instead and removed from project's root to avoid having the code out of sync with the build.

For now, consider cloning the repository and use the files directly as a better installation method 👍
